client/vnctl/lib/vnctl/cli/base.rb

```diff
@@ -242,7 +242,7 @@ def self.define_mode_relation(mode_type, required_opts = [], &block)
           }
 
           yield self if block_given?
-          desc "del #{base_uuid.upcase}_UUID" , "Removes a(n) #{mode_type} #{base_uuid}."
+          desc "del #{base_uuid.upcase}_UUID --option=OPTION", "Removes a(n) #{mode_type} #{base_uuid}."
```

Comment:
What is the `--option=OPTION` for? I don't think this command takes an --option argument.

```bash
> vnctl translations static add tr-q5z00rzz --ingress-ipv4-address 10.0.0.1 --egress-ipv4-address 10.0.0.5 --option joske

ERROR: vnctl add was called with arguments ["tr-q5z00rzz", "--option", "joske"]
Usage: "vnctl translations static_address add TRANSLATION_UUID --option=OPTION --egress-ipv4-address=EGRESS_IPV4_ADDRESS --ingress-ipv4-address=INGRESS_IPV4_ADDRESS".
```